[2024-10-17T21:15:46.610+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T21:15:46.613+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_branching.branch manual__2024-10-17T20:15:42.869534+00:00 [queued]>
[2024-10-17T21:15:46.614+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_branching.branch manual__2024-10-17T20:15:42.869534+00:00 [queued]>
[2024-10-17T21:15:46.614+0100] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-17T21:15:46.618+0100] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): branch> on 2024-10-17 20:15:42.869534+00:00
[2024-10-17T21:15:46.620+0100] {standard_task_runner.py:72} INFO - Started process 2795 to run task
[2024-10-17T21:15:46.623+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'executing_branching', 'branch', 'manual__2024-10-17T20:15:42.869534+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/branching.py', '--cfg-path', '/var/folders/nl/1x330z2x7yzb268yb0bbvjcr0000gn/T/tmp9fbpvgbs']
[2024-10-17T21:15:46.624+0100] {standard_task_runner.py:105} INFO - Job 242: Subtask branch
[2024-10-17T21:15:46.635+0100] {task_command.py:467} INFO - Running <TaskInstance: executing_branching.branch manual__2024-10-17T20:15:42.869534+00:00 [running]> on host mujtabas-macbook-pro.local
[2024-10-17T21:15:46.655+0100] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ali' AIRFLOW_CTX_DAG_ID='executing_branching' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2024-10-17T20:15:42.869534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-17T20:15:42.869534+00:00'
[2024-10-17T21:15:46.656+0100] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T21:15:46.660+0100] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-17T21:15:46.662+0100] {python.py:240} INFO - Done. Returned value was: eligible_to_drive
[2024-10-17T21:15:46.662+0100] {branch.py:38} INFO - Branch into eligible_to_drive
[2024-10-17T21:15:46.663+0100] {skipmixin.py:230} INFO - Following branch {'eligible_to_drive'}
[2024-10-17T21:15:46.664+0100] {skipmixin.py:278} INFO - Skipping tasks [('not_eligible_to_drive', -1)]
[2024-10-17T21:15:46.669+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T21:15:46.669+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_branching, task_id=branch, run_id=manual__2024-10-17T20:15:42.869534+00:00, execution_date=20241017T201542, start_date=20241017T201546, end_date=20241017T201546
[2024-10-17T21:15:46.685+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T21:15:46.692+0100] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-17T21:15:46.692+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
